updates/next.json

```diff
@@ -13,19 +13,22 @@
       }
     },
     {
-      "version": "32.20200923.1.0",
+      "version": "33.20201006.1.0",
       "metadata": {
         "rollout": {
           "start_percentage": 1.0
         }
       }
     },
     {
-      "version": "33.20201006.1.0",
+      "version": "33.20201020.1.0",
       "metadata": {
+        "barrier": {
+          "reason": "https://github.com/coreos/fedora-coreos-tracker/issues/646"
```

Comment:
wouldn't the barrier only be needed on the last release that includes the migration code?

Reply:
We're not planning to keep the migration code very long, so there's not a big difference between doing it now vs when we remove it. (I agree ideally though we'd do it on removal so that we can delay and combine barriers as much as possible. It's just more easy for us to get wrong.)